mdflynn/rancid-tomatillos
# Feature/unit testing

## Is this a fix or a feature?
This is a feature. 

## What is the change?
This adds unit and integration testing for our App.

## What does it fix?
No fix this pull request.

## Where should the reviewer start?
- Review each test file in its entirety.

## How should this be tested?
- Clone down the repo
- `cd` into the directory
- Run `npm test` to see passing test suite.
